src/main/java/com/mybookfinder/enterprise/dto/Book.java

```diff
@@ -3,6 +3,7 @@
 import lombok.Data;
 
 public @Data class Book {
+    private int bookId;
```

Comment:
Added a unique identifier variable for the book class. This will make it easy to find and track books as new ones are created.
Source: PlantPlaces lab and module 3 video on Lombok